src/v2/view-builder/components/ChallengeWebauthnFooter.js

```diff
@@ -0,0 +1,98 @@
+import AuthenticatorFooter from './AuthenticatorFooter';
+import { $, _, loc, View } from 'okta';
+import { getBackToSignInLink } from '../utils/LinksUtil';
+import Link from './Link';
+import hbs from 'handlebars-inline-precompile';
```

Comment:
style-nit: Module-like or alias imports usually are located at the top of the file. Can we keep the `okta` and `handlebars-inline-precompile` imports up top and together?